docs/ddl-operation-on-carbondata.md

```diff
@@ -220,13 +221,11 @@ of columns is used.
                                 saleQuantity Int,
                                 revenue Int)
    STORED BY 'carbondata'
-   TBLPROPERTIES ('COLUMN_GROUPS'='(productName,productCategory)',
+   TBLPROPERTIES ('COLUMN_GROUPS'='(productName,productNumber)',
```

Comment:
Column-groups should be contiguous as I stated in my PR description, it was giving exception that column-groups should be contiguous